<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KontierungDialog</class>
 <widget class="QDialog" name="KontierungDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="text">
      <string>&lt;h1&gt;Kontierung&lt;/h1&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelTime">
       <property name="text">
        <string>Time:</string>
       </property>
       <property name="buddy">
        <cstring>timeEditTime</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelTimespan">
       <property name="text">
        <string>Timespan:</string>
       </property>
       <property name="buddy">
        <cstring>timeEditTimespan</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelProjekt">
       <property name="text">
        <string>Projekt:</string>
       </property>
       <property name="buddy">
        <cstring>comboBoxProjekt</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelSubprojekt">
       <property name="text">
        <string>Subprojekt:</string>
       </property>
       <property name="buddy">
        <cstring>lineEditSubprojekt</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelWorkpackage">
       <property name="text">
        <string>Workpackage:</string>
       </property>
       <property name="buddy">
        <cstring>lineEditWorkpackage</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelText">
       <property name="text">
        <string>Text:</string>
       </property>
       <property name="buddy">
        <cstring>lineEditText</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QTimeEdit" name="timeEditTime">
       <property name="displayFormat">
        <string>HH:mm:ss</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTimeEdit" name="timeEditTimespan">
       <property name="displayFormat">
        <string>HH:mm:ss</string>
       </property>
       <property name="time">
        <time>
         <hour>1</hour>
         <minute>0</minute>
         <second>0</second>
        </time>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBoxProjekt"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEditSubprojekt"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEditWorkpackage"/>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEditText"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>KontierungDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>KontierungDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
